Documentation/operator.md

```diff
@@ -0,0 +1,46 @@
+Usage of Operator
```

Comment:
```suggestion
Operator CLI Flags
```

nit :)

Reply:
I think we also need the following at the top of the file so it can be included on website (similar to how `API.md` is included):
```
---
title: "Operator CLI Flags"
description: "Lists of possible arguments passed to operator executable."
lead: ""
date: 2021-06-18T08:49:31+00:00
draft: false
images: []
menu:
  docs:
    parent: "operator"
weight: 1000
toc: true
---
```

date can be static for now.

Reply:
modification done as instructed :)